File: Coding/03_Beginner/Full_search/C++/3.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int N, V;
    cin >> N >> V;
    vector<int> A(N);
    for (int i = 0; i < N; i++) cin >> A[i];

    bool flag = false;
    int ans = -1;

    for (int i = 0; i < N; i++){
        if(A[i]==V){
            ans = i;
            flag = true;
        }
    }
    if (flag){
        cout << ans << endl;
    }else{
        cout << -1 << endl;
    }

    return 0;
}

File: Coding/03_Beginner/Full_search/JavaScript/3.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const V = Number(input[1]);
const A = input.slice(2).map(Number);

let flag = false;
let asn = -1
for (let i = 0; i < N; i++){
    if (A[i] == V){
        ans = i
        flag = true
    }
}

if (flag){
    console.log(ans);
}else{
    console.log(-1);
}